lib/Agoneum/account/account.ex

```diff
@@ -63,7 +63,20 @@ defmodule Agoneum.Account do
   """
   def create_user(attrs \\ %{}) do
```

Comment:
Yep. So Account is in singular basically because that was the tense used in McCord's presentation. As it's the upper case, and the `User` module is also singular, I'd argue that 'Account` is fine. I really think this is pretty subjective though.